### DISCUSSION TITLE
defining gradient in theme

### DISCUSSION THREAD

**Discussion by CFarhad** [mantinedev/mantine#4838] [Q&A]
How can i define some gradients like colors in mantine theme.
Also how can i define a warning color with single color in theme without defining an array of colors for warning. only a hex color.

**Comment by rtivital** [comment]
> How can i define some gradients like colors in mantine theme.

Mantine theme does not support gradients, you can store them in CSS variables and use in styles.

> Also how can i define a warning color with single color in theme without defining an array of colors for warning. only a hex color.

https://mantine.dev/theming/theme-object/#other

**Reply by empz** [reply]
**UPDATE**

Nevermind, found this: https://mantine.dev/styles/css-variables/#css-variables-resolver

---

Could you please provide a full example? It's not clear how theme other objects map to CSS variables.

```ts
const theme = createTheme({
  other: {
    gradient: "linear-gradient(135deg, #f6eeff 0%, #cab1ea 100%)"
  }
});
```
And then?

```jsx
<div style={{ background: "var(--mantine-other-gradient)" }} />
```
Doesn't work

**Reply by rtivital** [reply]
This is how you can define CSS variables in your application – https://medium.com/@rvnranaivo/csss-root-variables-use-it-2f64bd6f1363
You do not need `theme.other` for that.
